### ISSUE TITLE
bug: unpronounceable controls  when using the app

### ISSUE THREAD

**Issue by marrie** [hariettem/Artemis#335]
**Describe the bug**
when  I am  in the main home tab I hear "unpronounceable" by voiceover. Clicking taht button brings me tgo a window with a cancel button in it as far as I can tell. I see no way to view the magazines I'm subscribed to from that menu, if that is what it is fore.

**To Reproduce**
'...'1. launch voiceover. This can be found in settings,  accessibility / vision / voiceover
2. launch the app
3. make sure you are at the main home screen.
This is when you can use explore by touch and or swiping to find the control mentioned in the first part of the report.

**Expected behavior**

I should be able to know what the controls do in the app.

**Screenshots**
![window that opens up once that button is tapped. Voiceover calls it a top tpday cancel something.](https://github.com/hariettem/Artemis/assets/3614487/94f92be1-8c60-41ac-9c50-317a2a38b104)
![area where the control is. It's in the home screen.](https://github.com/hariettem/Artemis/assets/3614487/c03e545e-322b-45f6-9e1c-06d13fd69c3c)

**Smartphone (please complete the following information):**
iPhone 13 pro
 - OS: iOs 17b3
0.1.7
**Additional context**

**Comment by hariettem** [comment]
fixed this for a bunch in the toolbar. Still need to keep finding more

**Comment by marrie** [comment]
Thanks so much. 👍

**Comment by RheingoldRiver** [comment]
I'm gonna close this ticket since the specific ones were found. @marrie can you open a new ticket if you find any new problems? You can be pretty brief in your ticket and title it "More unpronounceable controls" and provide a clear list of where to find them
